parts/k8s/kubernetesmaster-kube-apiserver.yaml

```diff
@@ -17,7 +17,8 @@ spec:
         - "--admission-control=NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,ResourceQuota"
         - "--address=0.0.0.0"
         - "--allow-privileged"
-        - "--insecure-port=8080"
+        - "--insecure-port=0"
```

Comment:
This interacted strangely with `/heatlhz` in k8s 1.8 on AKS. A random listen port was assigned and would later fail health /cc @khenidak 

We instead kept the port as `8080` and added `--insecure-bind-address=127.0.0.1`